src/backend/main/py.main/caliopen_main/user/core/user.py

```diff
@@ -334,6 +348,29 @@ def setup_system_tags(self):
             tag['date_insert'] = datetime.datetime.now(tz=pytz.utc)
             Tag.create(self, **tag)
 
+    def setup_settings(self):
+        """Create settings related to user."""
+        # XXX set correct values
```

Comment:
We will need to fill these defaults settings from config file.

Reply:
yes